ouroboros-consensus/ouroboros-consensus.cabal

```diff
@@ -284,6 +284,7 @@ library
                      , binary            >=0.8   && <0.11
                      , bytestring        >=0.10  && <0.11
                      , cardano-binary
+                     , cardano-ledger-alonzo
```

Comment:
I'm hoping that with my suggestion in this thread https://github.com/input-output-hk/ouroboros-network/pull/3224/files#r657368550 , we wouldn't need this dep here (since we'd no longer need `ExUnits` in the `LedgerSupportsMempool` declaration).